packages/components/src/components/badge/badge.scss

```
@@ -69,8 +75,7 @@
 		@extend %db-overwrite-font-size-sm;
 		padding-inline: variables.$db-spacing-fixed-xs;
```

```suggestion
		&:not(:empty,
		> span:empty) { padding-inline: variables.$db-spacing-fixed-xs; }
```

instead of overwriting the padding afterwards on line 79 and even also to prevent the need for a mixin.